<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_8">
        <property name="text">
         <string>iterate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_7">
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_4">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_6">
        <property name="text">
         <string>save as</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_5">
        <property name="text">
         <string>load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_vars">
       <attribute name="title">
        <string>Variables</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="collapseVariables">
            <property name="text">
             <string>collapse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="expandVariables">
            <property name="text">
             <string>expand</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>add var set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>remove var set</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTreeView" name="variablesView">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="iteratorsView"/>
          </item>
          <item>
           <widget class="QTreeView" name="fullView"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="addVariableButton">
            <property name="text">
             <string>add var</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="addVariableGroupButton">
            <property name="text">
             <string>add group</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>expandVariables</sender>
   <signal>clicked()</signal>
   <receiver>variablesView</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>collapseVariables</sender>
   <signal>clicked()</signal>
   <receiver>variablesView</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
